content/docs/accessibility.md

```diff
@@ -361,80 +360,81 @@ class BlurExample extends React.Component {
 }
 ```
 
-This code exposes the functionality to both pointer device and keyboard users. Also note the added `aria-*` props to support screen-reader users. For simplicity's sake the keyboard events to enable `arrow key` interaction of the popover options have not been implemented.
+Bu kod fonksiyonelliği, hem imleci hem de klavye kullanıcılarını açığa çıkartır. Ekran-okuyucu kullanıcılarını desteklemek için eklenen `aria-*` prop'larına da dikkat edin. Basitlik adına, açılır pencere (popover) seçeneklerinin `arrow key` etkileşimini etkinleştirmek için klavye olayları uygulanmamıştır.
```

Comment:
This should be better:

`Bu kod, işlevselliği hem imlece hem de klavye kullanıcılarına sunar.`

Reply:
This was my first choice, what I was told that not to use word of 'işlev'

Reply:
You can use `fonksiyonellik` or `islev`. That was not the problem in that sentence. 

I wanted to change the way you translate the word `expose`. Also, the first translation meant a totally different thing if you look at the original sentence.

Reply:
At the first look, I have seen the replaced word of `islevsellgini`. If it was not a problem to use either one, why did you change it? I thought, this was the change you were asking for. Your suggestion could be descriptive or to be targeted on the word what you wanted to change.
What might be the English word to your suggested word of `sunar`? `Provide`?.. 
`Provide` and `expose` have completely different meanings.
What is the meaning of `expose`? [Cambridge dictionary](https://dictionary.cambridge.org/dictionary/english-turkish/expose) and [Oxford dictionary](https://www.oxfordlearnersdictionaries.com/definition/english/expose_1?q=expose):`to show something hidden`
My sentence is grammatically wrong that gives different meaning because of wrong suffixes. It is not the way I translate the word of `expose`, it is the the way I translate the objects of  `pointer device` and `keyboard users`.
The way you translate the word of `expose` excludes the context of the document and the meaning of the original word by adding another meaning to make the sentence sounds well. 
I do not agree with your translation however, there is no need to keep this conversation going. I have to respect you time, changes you are asking will be applied. Thank you.